mediator/src/controllers/service-request.ts

```diff
@@ -19,18 +22,41 @@ async function createServiceRequest(request: ServiceRequest) {
       }
     };
 
-    const res = await axios.get(`${fhirUrl}/Patient/${patientId}`, options);
+    const res = await axios.get(`${fhirUrl}/Patient/?identifier=${patientId}`, options);
 
     if (res.status !== 200) {
       return {status: res.status, data: res.data};
     }
 
-    // todo => @njogz to add task creation process.
+    // generate subscription resource
+    const FHIRSubscriptionResource = generateFHIRSubscriptionResource(patientId, callbackURL);
```

Comment:
We had a discussion about intercepting the FHIR subscription at Zanzibar. Are we still implementing that?

Reply:
Oh, okay. Should I create an issue for that?